frontend/pages/review.tsx

```diff
@@ -106,136 +169,127 @@ function ApprovalList({ category, filter }: { category: ApprovalCategory; filter
-    return <Paper sx={{ mt: 2, mb: 2 }} />
-  }
-
   return (
-    <Paper sx={{ mt: 2, mb: 2, pb: 2 }}>
-      <Typography sx={{ p: 3 }} variant='h4'>
-        {category === 'Upload' ? 'Models' : 'Deployments'}
-      </Typography>
-      {approvals.map((approvalObj: any) => (
-        <Box sx={{ px: 3 }} key={approvalObj._id}>
-          <Grid container spacing={1} sx={approvalObj.approvalType === 'Manager' ? managerStyling : reviewerStyling}>
-            <Grid item xs={9}>
-              {category === 'Upload' && (
-                <>
-                  <Link href={`/model/${approvalObj.version?.model?.uuid}`} passHref legacyBehavior>
-                    <MuiLink
-                      variant='h5'
-                      sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
-                    >
-                      {approvalObj.version?.metadata?.highLevelDetails?.name}
-                    </MuiLink>
-                  </Link>
-                  <Stack direction='row' spacing={2}>
-                    <Chip color='primary' label={approvalObj.approvalType} size='small' />
-                    <Chip color='primary' label={`Version: ${approvalObj.version?.version}`} size='small' />
-                    <Box sx={{ mt: 'auto !important', mb: 'auto !important' }}>
-                      <Typography variant='body1'>
-                        {approvalObj.version?.metadata?.highLevelDetails?.modelInASentence}
-                      </Typography>
-                    </Box>
-                  </Stack>
-                  {approvalObj.version === undefined ||
-                    (approvalObj.version === null && (
-                      <Alert sx={{ mt: 2 }} severity='warning'>
-                        This model appears to have data missing - check with the uploader to make sure it was uploaded
-                        correctly
-                      </Alert>
-                    ))}
-                </>
-              )}
-              {category === 'Deployment' && (
-                <>
-                  <Link href={`/deployment/${approvalObj.deployment?.uuid}`} passHref legacyBehavior>
-                    <MuiLink
-                      variant='h5'
-                      sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
-                    >
-                      {approvalObj.deployment?.metadata?.highLevelDetails?.name}
-                    </MuiLink>
-                  </Link>
-                  <Typography variant='body1'>
-                    Requesting deployment of{' '}
-                    <Link href={`/model/${approvalObj.deployment?.model?.uuid}`} passHref legacyBehavior>
-                      <MuiLink sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}>
-                        {approvalObj.deployment?.model?.latestVersion?.metadata?.highLevelDetails?.name}
+    <>
+      {approval && (
```

Comment:
This check shouldn't be needed once you've addressed the following comment https://github.com/gchq/Bailo/pull/663/files?diff=unified&w=1#r1302024117